main backend file.py

```diff
@@ -0,0 +1,39 @@
+
+from bottle import Bottle, run, request, template
+
+
+app = Bottle()
+
+@app.route('/', method = 'POST')
+def post_redirect():

```

Comment:
I noticed your two method names are post_redirect and get_redirect.   Why did you name these methods using redirect, the methods aren't actually performing a redirect are they?

Reply:
I suppose you are correct about that. Since I'm not actually redirecting to a new address, the use of the phrase "redirect" is a misnomer. I'm actually using it to directly converse with the address I specified, so no redirection occurs.

I had named them using redirect because I had misread some code I had written in my masters course where we used "redirect". The actual code itself is different, I had just used the same name without thinking about it.